lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_betabuild_prechecks.rb

```diff
@@ -80,6 +81,11 @@ def self.available_options
                                        description: 'Skips confirmation',
                                        is_string: false, # true: verifies the input is a string, false: every kind of value
                                        default_value: false), # the default value if the user didn't provide one
+          FastlaneCore::ConfigItem.new(key: :default_branch,
+                                       env_name: 'FL_RELEASE_TOOLKIT_DEFAULT_BRANCH',
+                                       description: 'Default branch of the repository',
+                                       is_string: false, # true: verifies the input is a string, false: every kind of value
```

Comment:
Well this parameter is supposed to be a String, and shouldn't be a boolean, integer or whatnot. So better use validation here. You could use `is_string: true`, but in practice that `is_string` flag has been obsoleted in Fastlane core, and is now replaced by a more versatile `type:`, so better use that from now on.

```suggestion
                                       type: String,
```

(PS: I'm thinking at some point in the future we should probably go thru all the existing actions in the toolkit to replace all the ConfigItems for which we used `is_string: true` to use `type: String` instead, not because it would really matter or fix anything behavior-wise, but because most of us use copy/pasting when we add new items or create new actions, so we might as well set a good precedent. Obviously this is not something for this PR to tackle, but something to keep in mind for later)

Reply:
🤦

Reply:
Addressed in dea6cb49f561f39b5ab05edb9041175aaddc6e3e.